src/benchmarks/micro/coreclr/GC/Perf_GC.cs

```diff
@@ -0,0 +1,36 @@
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Reflection;
+
+namespace System.Tests
+{
+    [GenericTypeArguments(typeof(byte))]
+    [GenericTypeArguments(typeof(char))]
+    [BenchmarkCategory(Categories.CoreCLR)]
+    public class Perf_GC<T>
+    {
+        [Benchmark]
+        [Arguments(512)]
+        [Arguments(512 * 8)]
+        public T[] NewOperator_Array(int length) => new T[length];
+
+#if !NETFRAMEWORK && !NETCOREAPP2_1 && !NETCOREAPP2_2// API added in .NET Core 3.0 (internal)
```

Comment:
Should this use the conditional compilation strategy since we're adding a new file for this test?